.changelogs/410-reviews-avoid-use-of-inline-css.yml

```
@@ -0,0 +1,5 @@
+significance: patch
```

I'm not sure this is just a `patch`, it might have impacts on the frontend as we're moving some inline style into a stylesheet, which means less specificity.
```suggestion
significance: minor
```